CMakeLists.txt

```diff
@@ -10,6 +10,9 @@ set(Brion_VERSION_ABI 9)
 
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/CMake
   ${CMAKE_SOURCE_DIR}/CMake/common ${CMAKE_SOURCE_DIR}/Pydoxine/CMake)
+if(NOT EXISTS ${CMAKE_SOURCE_DIR}/CMake/common/Common.cmake)
+  message(FATAL_ERROR "CMake/common missing, run: git submodule update --init")
+endif()
 
 include(GitExternal)
```

Comment:
Actually it is, but Pydoxine. Unless we make this also a submodule, but then it'd be replicated by RTNeuron as we discussed.